### DISCUSSION TITLE
PoE anstelle von Wifi?

### DISCUSSION THREAD

**Discussion by Undead2211** [frickelzeugs/FingerprintDoorbell#5] [Ideas]
Hi,
geil geil geil. Ich hab dein Projekt bei Spiel und Zeug gesehen und bin von dem ganzen extrem "angefixt".

Hast du evtl. auch Pläne eine adapterPlatine mit einem RJ45 Jack zu machen/dokumentieren damit man das Ding anstelle von Wifi mittels PoE ins Netz hängen kann?
Das würde 2 Fliegen mit einer Klappe schlagen. Prinzipiell nichts gegen Wifi, aber Infrastrukturelle Dinge habe ich gern Wired, zum einen wegen der Zuverlässigkeit (Abschirmung, schlechter Empfang, Störung, Ausfall, etc...) zum anderen bräuchte man dann kein Netzteil mehr etc..

Wäre das viel Aufwand für dich bzw. hättest du Lust dazu?

Grüße
Steffen

**Comment by henfri** [comment]
The hardware is simple. You could buy a board like this one:
https://www.olimex.com/Products/IoT/ESP32/ESP32-POE/open-source-hardware

But you would have to adopt the software:
https://github.com/OLIMEX/ESP32-POE/blob/master/SOFTWARE/ESP-IDF/ESP32_PoE_Ethernet_IDFv4.0/main/ethernet_example_main.c

**Comment by hesne** [comment]
If someone has a solution to use poe instead of power + wifi, would be great you could share.

**Reply by mkrasselt1** [reply]
Easyer Solution: wt32-eth01 bopard and a poe splitter for 5V
https://www.roboter-bausatz.de/p/wt32-eth01-esp32-modul-mit-ethernet-bluetooth-wifi
AND
https://www.amazon.de/DSLRKIT-Active-Splitter-Ethernet-Raspberry/dp/B01H37XQP8
 for example

**Reply by mkrasselt1** [reply]
And I would always recommend to use this approach over a dedicated poeboard because this way you could extract 12 V to power motors or other power hungry equipment

**Reply by hesne** [reply]
Thank you. But unfortunately I do not have the competence to modify the source code by myself.

**Reply by mkrasselt1** [reply]
If you are patient, I am in the progress of porting it myself...  But it will take some time

**Reply by hesne** [reply]
Yes. Thank you. Problem with this adapter: It is getting bigger and bigger what we have to bring under the surface. Just a PoE board is so much thinner

**Reply by mkrasselt1** [reply]
But, if this board is able to open your door - everything but the fingerprint reader should remain inside your house

**Reply by hesne** [reply]
So you would extend the fingerprint cables and put the devices inside? I have no idea where to put all this close to the door...

**Reply by Thomas1998XD** [reply]
How far did you get?  @mkrasselt1

**Comment by henfri** [comment]
I do Not understand your 2nd Point 
The Poe thing is 5v only?

**Reply by mkrasselt1** [reply]
Yeah, 5v only because the wt32-eth01 runs with usb - 5V and you won't need a buck converter to power the esp board.
But for Relais and Co you will need more than that.
I am working on a board that includes all of this

**Comment by dsteinkellner-a11y** [comment]
Hi everyone, I’d also love to switch from Wi-Fi to LAN. Using LAN would allow me to put the device into a VLAN for better security.
If anyone has the time to adapt the project for an ESP32-ETH01, that would be awesome. I spent the weekend trying to port it myself but ran into several errors in VS Code. To be honest, I’m also struggling with the flashing process for the ETH01.
If you have any better hardware recommendations for Ethernet support, I’d be very grateful!
Thanks,
Daniel

**Reply by Steinii50354** [reply]
Update, I managed to flash the ETH01.
